### PR TITLE
UsageStats: Add database age and driver

### PR DESCRIPTION
**Pull request by ryantxu**
**What is this feature?**

This adds the database age and driver to usage stats:
image

**Why do we need this feature?**
This will help us better understand the databases in the wild

### CODE REVIEW

File: pkg/services/stats/statsimpl/stats.go [hunk 1]
```diff
@@ -140,6 +140,17 @@ func (ss *sqlStatsService) GetSystemStats(ctx context.Context, query *stats.GetS
 
 		return nil
 	})
+
+	if err == nil {
+		result.DatabaseDriver = dialect.DriverName()
+		ts := time.Time{}
+		err = ss.db.GetSqlxSession().Get(ctx,
```
Comment (ryantxu) [inline pkg/services/stats/statsimpl/stats.go:147]: I could not get a reasonable behavior with the massive xorm query above, so I made a new sqlx query.  Any better suggestions?

File: pkg/infra/usagestats/statscollector/service.go [hunk 2]
```diff
@@ -161,6 +161,10 @@ func (s *Service) collectSystemStats(ctx context.Context) (map[string]interface{
 	m["stats.active_data_keys.count"] = statsResult.ActiveDataKeys
 	m["stats.public_dashboards.count"] = statsResult.PublicDashboards
 	m["stats.correlations.count"] = statsResult.Correlations
+	m["stats.database.created.time"] = statsResult.DatabaseCreatedTime
```
Comment (ryantxu) [inline pkg/infra/usagestats/statscollector/service.go:164]: @DanCech is there a better way to report this metric?   maybe age?

Comment (DanCech) [reply pkg/infra/usagestats/statscollector/service.go]: I think created time as a timestamp is fine, I also switched up the driver to just report the string directly

File: pkg/services/stats/statsimpl/stats.go (hunk 1 above)
Comment (DanCech) [reply pkg/services/stats/statsimpl/stats.go]: got it working in the main query

**Review by DanCech — APPROVED** [review]
I would have sworn I've seen lint change spacing in the other direction (like on line 125-126 !) but LGTM

**Pull request merged**